chapter02/worksheets/solution2.1.7.9_as_function.scala

```diff
@@ -14,6 +14,7 @@ Hint: use groupBy, map, sum.
 def totalCount(arg: Seq[(String, Int)]) = {
   // val a: Seq[(String, Int)] =  Seq(("pear", 2), ("apple", 2), ("apple", 6), ("pear", 1), ("lemon", 4), ("apple", 2), ("apple", 2))
   arg.groupBy(s => s._1).map { case (x, y) => (x, y.map(_._2).sum) }
+}
```

Comment:
Would you like to add some `assert` as a test for this function, instead of the comment below with `res37`?

Reply:
Sure. Will look for it.

Reply:
Still need to add `assert` here, or else the tests are not checking this part.